rclpy/rclpy/node.py

```diff
@@ -1019,11 +1040,11 @@ def set_parameters_callback(
         """
         Register a set parameters callback.
 
-        Calling this function with override any previously registered callback.
+        Calling this function will override any previously registered callback.
```

Comment:
The description seem to not match the actual behavior of the modified function anymore?